### DISCUSSION TITLE
Support for Signify Netherlands B.V.

### DISCUSSION THREAD

**Discussion by pnm1977** [Koenkk/zigbee2mqtt#13540] [Devices]
Are there plans to add support for devices by Signify Netherlands B.V.? 
I've built up quite a few Philips Hue / Signify devices and just started migrating to Zigbee2MQTT from ZHA. Was hoping to take advantage of the OTA feature in Z2M but doesn't look like the Signify devices are compatible.

**Comment by pnm1977** [comment]
Current device models I have for Signify -
LOM003
LOM006
LTA004
LTW012
LTW013
LWA024
LWB010
RWL022

**Reply by sjorge** [reply]
[LOM003](https://github.com/Koenkk/zigbee-herdsman-converters/blob/804429a69f2fc155f82393aa652f62843af3cdbb/devices/philips.js#L1926-L1939)
[LOM006](https://github.com/Koenkk/zigbee-herdsman-converters/blob/804429a69f2fc155f82393aa652f62843af3cdbb/devices/philips.js#L1968-L1981)
[LTA004](https://github.com/Koenkk/zigbee-herdsman-converters/blob/804429a69f2fc155f82393aa652f62843af3cdbb/devices/philips.js#L697-L703)
[LTW012](https://github.com/Koenkk/zigbee-herdsman-converters/blob/804429a69f2fc155f82393aa652f62843af3cdbb/devices/philips.js#L921-L927)
[LTW013](https://github.com/Koenkk/zigbee-herdsman-converters/blob/804429a69f2fc155f82393aa652f62843af3cdbb/devices/philips.js#L942-L948)
LWA024
[LWB010](https://github.com/Koenkk/zigbee-herdsman-converters/blob/804429a69f2fc155f82393aa652f62843af3cdbb/devices/philips.js#L753-L759)
[RWL022](https://github.com/Koenkk/zigbee-herdsman-converters/blob/804429a69f2fc155f82393aa652f62843af3cdbb/devices/philips.js#L1727-L1746)

Seems like most of them are support, just LWA024 is not, but it's probably not hard to get support added.

**Reply by pnm1977** [reply]
Hi. Thanks for your reply and explanation, much appreciated.
Typically, the bulb I tried was the LWA024 so I took the failed attempt along with the inability to search by "Signify" as meaning that all Signify devices were not supported. This bulb is the last one I received to test so makes sense that it's not available yet.
I'll have a go at the rest of them.

**Reply by bullmastiffo** [reply]
@pnm1977 did you manage to get it working? I've just bought a pair of LWA024, and I would like to know if I should start adding it from scratch

**Reply by pnm1977** [reply]
Hi. I've not tried with the LWA024 bulb. I left it connected through ZHA.

**Reply by bullmastiffo** [reply]
I've added the device https://github.com/Koenkk/zigbee-herdsman-converters/pull/6510
